docs/react-native/docs/triggers.md

```diff
@@ -1,8 +1,100 @@
 ---
-title: Trigger
-description: Trigger notifications to appear under specific commands such as scheduling.
+title: Triggers
+description: Understand how to use triggers to create notifications that fire under specific conditions.
 next: /react-native/docs/integrations/fcm
 previous: /react-native/docs/events
 ---
 
-This documentation is a work in progress.
+Triggers can be used to display notifications in-advance when a specific condition is met such as time or location.
```

Comment:
`or location.` Is it worth removing this until its supported?

Reply:
I mention it below it's not supported but yeah maybe if it's confusing